### PR TITLE
Duration arithmetic

### PR DESCRIPTION
**Pull request by ptomato**
Closes: #408

### CODE REVIEW

File: docs/balancing.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+The usual `constrain` and `reject` modes are not available.
+
+The default is `balanceConstrain` mode.
+The `balance` mode is only provided for convenience, since the following code snippets give the same result:
+
+```javascript
+duration3 = duration1.minus(duration2, { disambiguation: 'balance' });
+
+tmp = duration1.minus(duration2);
+duration3 = tmp.with(tmp, { disambiguation: 'balance' });
```
Comment (Ms2ger) [inline docs/balancing.md]: One more for `from`

**Comment by Ms2ger** [comment]
Travis issue is out of memory dealing with the test262 results. Looking for a solution.

**Comment by Ms2ger** [comment]
Rebasing over https://github.com/tc39/proposal-temporal/pull/491 should help.

**Comment by ptomato** [comment]
I have still mysteriously lost my merging privileges, but I do think I've addressed your comments from last round. Thanks for investigating the OOM error, I had overlooked it!

**Pull request merged**